editor/editor_node.h

```diff
@@ -741,6 +741,7 @@ class EditorNode : public Node {
 	static EditorFolding &get_editor_folding() { return singleton->editor_folding; }
 
 	static EditorTitleBar *get_title_bar() { return singleton->title_bar; }
+	static HBoxContainer *get_title_bar_right_hbox() { return singleton->title_right_hbox; }
```

Comment:
This was added to keep the main screen buttons centered after adding editor plugins to the title bar using `add_control_to_container(EditorPlugin.CONTAINER_TOOLBAR, Control)`.